content/stub.xhtml

```diff
@@ -837,7 +837,7 @@
             <span title="{{fullDate}}">{{date}}</span>
           </span>
           <span class="replyLinkWrapper">
-            <a href="javascript:" class="link replyMainActionLink" style="display:none"></a>
+            | <a href="javascript:" class="link replyMainActionLink"></a>
```

Comment:
It had been there before and was removed in https://github.com/protz/thunderbird-conversations/pull/1157/commits/63a6828f163cc97e6ac3e762db18fd1f7650639a. Apart from that, yes, I think this is a simple text node and totally valid.